backend/src/main/java/ai/verta/modeldb/batchProcess/DatasetToRepositoryMigration.java

```diff
@@ -337,13 +333,15 @@ private static void migrateDatasetCollaborators(String datasetId, Dataset datase
             .getCollaboratorType()
             .equals(CollaboratorTypeEnum.CollaboratorType.READ_WRITE)) {
           roleService.createRoleBinding(
-              readOnlyRole,
+              ModelDBConstants.ROLE_REPOSITORY_READ_ONLY,
+              null,
```

Comment:
Optional<> ????

Reply:
Optional shouldn't be used as a request parameter in java

Reply:
> Optional shouldn't be used as a function parameter in java

The team in charge of best practices at Twitter (and I) would disagree with you =) Any instance of `null` is asking for a null pointer exception.

Reply:
This parameter is not used anywhere except with the null value, so I removed it now

Reply:
Even better!